doc/requirements.txt

```
@@ -11,30 +11,29 @@ numpy
 pygments-github-lexers
 packaging
 scipy
-docutils==0.16
-sphinx~=3.5.0; python_version < "3.10"
-sphinx==4.2; python_version == "3.10"
+docutils==0.20
+sphinx==8.1
 sphinx-automodapi==0.13
 sphinx-copybutton
-sphinxcontrib-bibtex==2.4.2
-sphinxcontrib-applehelp==1.0.4
-sphinxcontrib-devhelp==1.0.2
-sphinxcontrib-qthelp==1.0.3
-sphinxcontrib-htmlhelp==2.0.1
-sphinxcontrib-serializinghtml==1.1.5
+sphinxcontrib-bibtex==2.6.3
+sphinxcontrib-applehelp==2.0.0
+sphinxcontrib-devhelp==2.0.0
+sphinxcontrib-qthelp==2.0.0
+sphinxcontrib-htmlhelp==2.1.0
+sphinxcontrib-serializinghtml==2.0.0
 tensorflow-cpu==2.18.0; platform_machine == "x86_64"
 tensorflow_macos==2.9.0; sys_platform == "darwin" and platform_machine == "arm64"
 tensornetwork==0.3
 tomlkit
-torch==2.3.0
-jinja2==3.0.3
+torch==2.5.1
+jinja2>=3.1
 rustworkx>=0.14.0
-networkx==2.6
-requests~=2.28.1
+networkx==3.1
+requests==2.32.3
 # we do not pin the sphinx theme, to allow the
 # navbar and header data to be updated at the source
 pennylane-sphinx-theme
 tomli~=2.0.0 # Drop once minimum Python version is 3.11
-sphinxext-opengraph==0.6.3 # Note: latest version 0.9.0 requires Sphinx >=4.0
+sphinxext-opengraph==0.9.0 # Note: latest version 0.9.0 requires Sphinx >=4.0
```

```suggestion
sphinxext-opengraph==0.9.0
```

Not sure if we need that comment anymore since we are sphinx 8+ after this